sync/src/synchronizer/mod.rs

```diff
@@ -592,7 +598,7 @@ mod tests {
             .store()
             .get_block_header(&shared.store().get_block_hash(number - 1).unwrap())
             .unwrap();
-        let parent_epoch = shared.get_block_epoch(&parent.hash()).unwrap();
+        let parent_epoch = shared.store().get_block_epoch(&parent.hash()).unwrap();
         let epoch = shared
```

Comment:
Please use `shared.get_block_epoch()`, there is a bit of nuance.

Reply:
don't use ChainProvider::get_block_epoch, it will be deleted later